reports/_input.qmd

```diff
@@ -10,7 +10,7 @@
 
 This tests if our Home Improvement Exemption characteristics are getting accurately updated.
 
-```{r}
+```{r hie_characteristics}
```

Comment:
**suggestion (blocking):** Let's actually prefix the chunk names with the name of the document. This will let us see which document a chunk is in in case of failure during the main render. For example:

```suggestion
```{r _input_hie_characteristics}
```